build_mime_types/README.md

```diff
@@ -0,0 +1,9 @@
+# build-mime-types
+
+Builds a class file for ContentType enum from the IANA mime types list.
```

Comment:
Where would the code live if it doesn't live in this repo?

```since the process of extending will be identical (update csv of the actual python enum file)```

Process:

1. Download CSV files
2. Run build_mime_types.py

If you delete `build_mime_types.py` you'll have to add thousands of enum lines by hand or do a manual diff. Maybe I'm missing somthing?

Reply:
In this case your suggestion with enum map for existing values would work fine as well, so it is up to you: update it now or leave PR as is and wait till the major version update w/o BC (but I have no time estimate on that for now)